src/data/finsupp/basic.lean

```diff
@@ -1077,6 +1077,13 @@ finset.subset.antisymm
   (calc support f = support (- (- f)) : congr_arg support (neg_neg _).symm
      ... ⊆ support (- f) : support_map_range)
 
+lemma support_sub [add_group G] {f g: α →₀ G} :
```

Comment:
This is missing
```suggestion
lemma support_sub [decidable_eq α] [add_group G] {f g : α →₀ G} :
```
which is why you had the awkward `convert` in the other file. It's needed by `∪` on `finset α`